memo/tests/test_my_learning_api.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+from rest_framework import status
+from decouple import config
+from model_bakery import baker
+from utils import convert_timestamp
+from memo.models import Card
+
+
+class MyLearningResultsTest(APITestCase):
+    def setUp(self):
+        self.User = get_user_model()
+        self.user = self.User.objects.create_superuser(
+            username="user", email="user@user.com", password="123"
+        )
+
+        token, created = Token.objects.get_or_create(user=self.user)
+        self.assertTrue(created)
```

Comment:
You don't need to put a assertion on the `setUp` method =]. 
Obs: the `_` is a way to tell python that the variable is not going to be used.
```suggestion
        token, _ = Token.objects.get_or_create(user=self.user)
```